Sprint-1/implement/max.js

```diff
@@ -1,4 +1,16 @@
 function findMax(elements) {
+  let maxNum = -Infinity;
+
+  if(elements.length === 0) {
+    return -Infinity;
+  }
+
+  for (element of elements) {
+    if(typeof element === 'number' && element > maxNum) {
+      maxNum = element;
+    }
+  }
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
findMax([-Infinity])
findMax([-Infinity, 0])
findMax([NaN])
findMax([0, NaN, 1])
```